src/components/DateRangePickerInput.jsx

```diff
@@ -27,6 +27,9 @@ import {
 
 const propTypes = forbidExtraProps({
   ...withStylesPropTypes,
+
+  children: PropTypes.node,
```

Comment:
do we need to allow any kind of children here?

Reply:
Yeah, let's restrict this to the DayPicker, DayPickerRangeController, and DayPickerSingleDateController. We can open it up if people ask.

Reply:
I think that it needs to be in DatePickerController as well, or else we'll need to move around         the logic for `this.maybeRenderDayPickerWithPortal()`

Reply:
Sorry @monokrome, can you clarify what you mean here?

Reply:
@majapw Sure! I think that ` this.maybeRenderDayPickerWithPortal() ` is rendered as a child of the input component right now. We can refactor to change this, but it would require additional effort:

https://github.com/airbnb/react-dates/blob/21e4a37a0ff02167d882a71137a8e8bfd1daa22c/src/components/SingleDatePicker.jsx#L568